src/app/components/observation/observation/observation.component.ts

```diff
@@ -245,13 +254,17 @@ export class ObservationComponent {
     return promise;
   }
 
-  //TODO: Midlertidig løsning for å åpne detaljert snøprofil og se større bilder direkte fra karusellen.
-  //På sikt skal nok klikk på bildet i karusell ta deg via en modal før du går videre til detaljert profil
-  imageClicked(attachment: AttachmentViewModel) {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const url = (attachment as any).Href || attachment.Url;
-    //kun snøprofil-bilder har href. Href er link til detaljert snøprofil. Andre biler åpnes i maks størrelse
-    window.open(url, '_blank');
+  async imageClicked(index: number) {
```

Comment:
Et mer beskrivende navn på denne metoden er vel kanskje `openImageCarousel` ?
```suggestion
  async openImageCarousel(index: number) {
```
Må i så fall fikses i template også..